requirements/install

```diff
@@ -0,0 +1,68 @@
+# Please make sure to cap all dependency versions, in order to avoid unwanted
+# functional and integration breaks caused by external code updates.
+#
+# These are general dependencies, so:
+#
+# * the ranges should be broad, and
+# * the number of them kept discreet.
+#
+# If a dependency seems trivial, please propose a pull request to remove it.
+#
+# General Rule
+# ========
+#
+# * Cap to latest major.
+#
+# NumPy Rule
+# ========
+#
+# * Cap to latest four minors (document why).
```

Comment:
This seems to be documented below already, appropriately at the `numpy` declaration line. Remove this part?